src/Checker.elm

```diff
@@ -0,0 +1,62 @@
+module Checker exposing (getIssues, getIssuesJSON, getIssuesString)
+
+import Configuration
+import Constraints
+import Encoders
+import Issue
+import Json.Encode
+import Models
+
+
+getIssues : Models.ParsedModule -> Configuration.Model -> List Issue.Issue
```

Comment:
Shouldn't this function be documented? :)

Reply:
Could you please also document all the other functions and their parameters?

Reply:
@mristin  I'll add compliance to elm-docstyle to the precommit checks :)